runtime/Python2/src/antlr4/TokenStreamRewriter.py

```diff
@@ -133,7 +135,7 @@ def getText(self, program_name, interval):
 
         if stop == len(self.tokens.tokens) - 1:
             for op in indexToOp.values():
-                if op.index >= len(self.tokens.tokens) - 1: buf.write(
+                if op.index > len(self.tokens.tokens) - 1: buf.write(
```

Comment:
This is a bug. If I insert something after the last token, that ``insertation`` will be executed twice!

A minimal working example is:

test.g4
```
grammar test;
ruleABC : 'aa' EOF ;
```

test.txt
```
aa
```

test.py
```
import sys
from antlr4 import *
from testLexer import testLexer
from testParser import testParser
from antlr4.TokenStreamRewriter import TokenStreamRewriter
from collections import namedtuple
from testListener import testListener
Interval = namedtuple("Interval", ["start", "stop"])

class MyListener(testListener):

    def __init__(self, tokens:BufferedTokenStream) -> None:
        super().__init__()
        self.writer = TokenStreamRewriter(tokens)

    def getText(self):
        return self.writer.getText(TokenStreamRewriter.DEFAULT_PROGRAM_NAME, Interval(0, len(self.writer.tokens.tokens)))

    def enterRuleABC(self, ctx:testParser.RuleABCContext):
        self.writer.insertAfterToken(ctx.stop, 'it appears first!\n')


def main(argv):
    input = FileStream('test.txt' if len(argv) <= 1 else argv[1])
    lexer = testLexer(input)
    stream = CommonTokenStream(lexer)
    parser = testParser(stream)
    tree = parser.ruleABC()
    myListener = MyListener(stream)
    walker = ParseTreeWalker()
    walker.walk(myListener, tree)

    print(myListener.getText())


if __name__ == '__main__':
    main(sys.argv)
```

the output is : 
```
aait appears first!
it appears first!
```

The purpose here is to insert **one** sentence. However, it ends up that **two** sentences are inserted!